lisa/tests/scheduler/eas_behaviour.py

```diff
@@ -869,9 +869,9 @@ def test_task_placement(self, energy_est_threshold_pct=18, nrg_model: EnergyMode
             capacity_margin_pct=capacity_margin_pct)
 
     @classmethod
-    def _get_rtapp_profile(cls, plat_info):
+    def _get_rtapp_profile(cls, plat_info, **profile_kwargs):
```

Comment:
You can just add `big_task_duty_cycle=Non`, there is no need for `**` as we are not trying to transparently forward the kwargs